spec.bs

```diff
@@ -1921,6 +1928,7 @@ and a [=real time reporting contributions map=] |realTimeContributionsMap|:
               |componentAuctionExpectedCurrency|, |topLevelOrigin|, and |realTimeContributionsMap|.
   1. Decrement |pendingBuyers| by 1.
 1. Wait until both |pendingBuyers| and |pendingAdditionalBids| are 0.

```

Comment:
I think the wait until pendingAdditionalBids may deadlock here, and not notice the abort?